pkg/config/config.go

```diff
@@ -51,5 +59,7 @@ func LoadConfig(path string, repoRootPathConfig string) (*Spec, error) {
 		return nil, fmt.Errorf("error unmarshaling config: %w", err)
 	}
 
+	fmt.Printf("Loaded config %v", config)
```

Comment:
nit: This is unnecessary and it'd be abit confusing to see the raw config output from the struct in the output - we can revisit more logging when we're closer to feature complete

Reply:
leftover from our debugging session. 😅